jboss/container/wildfly/galleon/fp-content/java/module.yaml

```diff
@@ -6,3 +6,7 @@ description: java content to add to fp.
 execute:
 - script: configure.sh
   user: 185
+  
+modules:
```

Comment:
This was already merged in https://github.com/wildfly/wildfly-cekit-modules/pull/70/files